datadog_lambda/trigger.py

```diff
@@ -299,6 +299,11 @@ def extract_http_tags(event):
     if headers and headers.get("Referer"):
         http_tags["http.referer"] = headers.get("Referer")
 
+    # Try to get `routeKey` from API GW v2; otherwise try to get `resource` from API GW v1
+    route = event.get("routeKey") or event.get("resource")
+    if route:
```

Comment:
Do we really need to define a variable here?

Reply:
I'm pretty sure we could just have a `None` tag and it should be ignored by the backend, but I'm not sure how risky is that, I'd check the tracer

Reply:
We have to parse the route now, so I think it makes sense to leave the variable now.
`http_tags["http.route"] = route.split(" ")[-1]`